omniledger/service/transaction.go

```diff
@@ -109,6 +143,11 @@ type Argument struct {
 // Arguments is a searchable list of arguments.
 type Arguments []Argument
 
+// NewArguments is a convenience method to create Arguments.
```

Comment:
Not a super fan of convenience methods that fit into one line. How inconvenient is it really to just say what it is in the calling code?

Smaller APIs are better, if we can manage it.